src/gcp_scanner/crawl.py

```diff
@@ -312,12 +299,8 @@ def get_firewall_rules(
     request = compute_client.firewalls().list(project=project_name)
     while request is not None:
       response = request.execute()
-      for firewall in response.get("items", []):
-        firewall_rules_list.append((
-            firewall["name"],
-            # firewall['description'],
-        ))
-
+      firewall_rules_list=list(map(lambda x:(x["name"],),
```

Comment:
Why did you change this from the list comprehension to this? This is less readable. You were only missing a comma in the previous version.

Reply:
reverted the map() to list comprehension